### PR TITLE
refactor: CLI command improvement (#1818)

### PR DESCRIPTION
**Pull request by yinheli**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?

[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #1818

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

**Comment by yinheli** [comment]
Okay, I'll resolve conflicting.

File: client/src/ctx.rs [hunk 1]
```diff
@@ -325,18 +326,44 @@ impl SessionContext {
...
+        let mut reader = std::io::BufReader::new(file);
+        let pb = progress_bar::new_with_size(size);
+
+        let (tx, mut rx) = mpsc::channel::<String>(10);
+
+        let producer = tokio::spawn(async move {
+            let mut buffer = String::new();
+            let mut line_count = 0;
+            while let Ok(size) = reader.read_line(&mut buffer) {
+                if size == 0 {
```
Comment (zipper-meng) [inline client/src/ctx.rs:339]: Why not put the check to see if the file is empty right after the file is opened?

```rust
let size = file.metadata()?.len();
if size == 0 {
    return Ok(ResultSet::Bytes(("".into(), 0)));
}
```

**Review by zipper-meng — DISMISSED** [review]
It's good.

**Review by zipper-meng — APPROVED** [review]
It's well.

**Pull request merged**